package.json

```diff
@@ -44,5 +44,8 @@
     "rxjs": "^6.3.3",
     "typescript": "^3.1.1",
     "uglify-js": "^3.4.9"
+  },
+  "dependencies": {
+    "release": "^6.0.0"
```

Comment:
Why do we need release?

Reply:
There, I dont' know. I've had the same question, I was certain it was already here... I remove that. 😱